content/config_rb_client.md

```diff
@@ -464,11 +469,11 @@ This configuration file has the following settings:
 
 `node_name`
 
-:   The name of the node. Determines which configuration should be
+:   The name of the node. This determines which configuration should be
     applied and sets the `client_name`, which is the name used when
-    authenticating to a Chef Infra Server. The default value is the Chef
-    Infra Client FQDN, as detected by Ohai. In general, Chef recommends
-    that you leave this setting blank and let Ohai assign the FQDN of
+    authenticating to a Chef Infra Server. By default, Chef Infra Client
+    will use the system's FQDN as the node name. In general, Chef recommends
+    that you leave this setting blank and let the client assign the FQDN of
     the node as the `node_name` during each Chef Infra Client run.
```

Comment:
Feels like there should be something here about the "unique identity" of the node.

Reply:
Fixed everywhere